README.md

```diff
@@ -172,6 +172,20 @@ clang-format can be installed by command:
 - macOS: `$ brew install clang-format`
 
 
+## Usage
+`Tangle-accelerator` currently supports two kinds of APIs
+* direct API: check [wiki page](https://github.com/DLTcollab/tangle-accelerator/wiki) for details.
+* IRI Proxy API
+
+### IRI Porxy API
+`tangle-accelerator` allow the use of IRI core APIs. User doesn't need to know whether there target machine is running a IRI or not. With the exactly same format of IRI API, `tangle-accelerator` would help user pass the request to IRI and pass the the response back to user. The followings are currently supporting Proxy APIs:
```

Comment:
Shall be "allows".

Reply:
Change the statement "User doesn't need to know whether there target machine is running a IRI or not" to 
> The calling process does not have to be aware of the destination machine running IRI.

Reply:
The term shall be "forward" rather than "pass". The action "forward" is to disclose original information of a client connecting to Tangle through proxy APIs.